requirements-dev.txt

```diff
@@ -379,7 +369,7 @@ semver==2.13.0
     # via
     #   -r requirements.txt
     #   prance
-sentry-sdk[flask]==1.9.9
+sentry-sdk[flask]==1.31.0
```

Comment:
As discussed with Sarah - this needs removing from notifications, utils version needs a release first

Reply:
resolved

Reply:
requitements.text does not need removing as it imports the sentry-sdk from utils

Reply:
See comment in next resolved comments